Article/Article.js

```diff
@@ -3,27 +3,48 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton= this.domElement.querySelectorAll(".expandButton");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.forEach((button) => {
```

Comment:
this.expandButton.innerText = 'expand' 'would also serve the same functionality'